### PR TITLE
Create operatingsystems-linux-ssh Monitoring proc V2 EN

### PR DESCRIPTION
**Pull request by SofieneB**

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+| Mandatory   | Name            | Description                                                                                 |
+| :---------- | :-------------- | :------------------------------------------------------------------------------------------ |
+| X           | SSHBACKEND      | Name of the backend: ```libssh```                                                           |
+| X           | SSHUSERNAME     | By default, it uses the user running process ```centengine``` on your poller                |
+|             | SSHPASSWORD     | Can be used. If not set, SSH key authentication is used                                     |
+|             | SSHPORT         | By default: 22                                                                              |
+|             | SSHEXTRAOPTIONS | Customize it with your own if needed. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```          |
+
+With that backend, you don't have to validate the target server fingerprint manually. Nice ;)
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:313]:
```suggestion
With that backend, you don't have to validate the target server fingerprint manually.
```

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+	
+	
+OK: CPU(s) average usage is 11.91 % - CPU '0' usage : 11.91 % |
+'cpu.utilization.percentage'=11.91%;;;0;100 '0#core.cpu.utilization.percentage'=11.91%;;;0;100 
+CPU '0' usage : 11.91 %
+```
+
+The command above gets the average of a Linux SSH CPU (```--mode=CPU```).
+
+This command will trigger a WARNING alarm if the CPU Average increases to more than 60% (```--warning-average='60'``)
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:346]:
```suggestion
This command will trigger a WARNING alarm if the CPU Average increases to more than 60% (```--warning-average='60'```)
```

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+
+3. On the Centreon Web interface in "Configuration > Plugin packs > Manager", install the *Linux SSH* Plugin-Pack
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Configuration
+
+Adding a new host into Centreon, apply the relevant host template matching your instance/cluster type. All of the host templates begin with *Os-Linux-SSH-custom*.
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:275]:
```suggestion
.Add a new Host and apply the Os-Linux-SSH-custom Host Template. 
```

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+---
+
+## Overview
+
+Linux is a family of open source Unix-like operating systems based on the Linux kernel, an operating system kernel first released on September 17, 1991, by Linus Torvalds. 
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+This Plugin-Pack provides assets to monitor all types of Linux based systems with a SSH server enabled:
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:13]:
```suggestion

This Plugin-Pack provides assets to monitor all types of Linux based systems with a SSH server enabled:
```

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+| cpu.utilization.percentage         | CPU utilization. Unit : %         |
+| core.cpu.utilization.percentage    | CPU utilization by core. Unit : % |
+
+
+<!--Cpu Detailled-->
+
+| Metric name           | Description                             |
+| :-------------------- | :-------------------------------------- |
+| cpu                   | Average usage for each CPUs. Unit : %   |
+| total_cpu             | Average total CPUs. Unit : %            |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:38]: Missing perfdata for cpu-detailed ?

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+<!--Diskio-->
+
+| Metric name                            | Description                                                  |
+| :------------------------------------- | :----------------------------------------------------------- |
+| device.io.read.usage.bytespersecond    | Read IO usage in Bytes per second. Unit : B/s                |
+| device.io.write.usage.bytespersecond   | Write IO usage in Bytes per second. Unit : B/s               |
+| device.io.read.time.milliseconds       | Read time in milliseconds. Unit : ms                         |
+| device.io.write.time.milliseconds      | Write time in milliseconds. Unit : ms                        |
+| device.io.write.time.milliseconds      | Write time in milliseconds. Unit : ms                        |
+| device.io.utils.percentage             | IO utilization for different usage type of CPU. Unit : ms   |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:75]:
```suggestion
| device.io.utils.percentage             | IO utilization for different usage type of CPU. Unit : %   |
```

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 7]
```diff
@@ -0,0 +1,346 @@
...
+| swap.usage.percentage       | Swap usage in percentage. Unit: % |
+
+
+<!--Load-->
+
+| Metric name                 | Description                           |
+| :-------------------------- | :------------------------------------ |
+| load1                       | Load average on 1 minute. Unit : %    |
+| load5                       | Load average on 5 minutes. Unit : %   |
+| load15                      | Load average on 15 minutes. Unit : %  |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:100]: The load is not a percentage ;-)

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 8]
```diff
@@ -0,0 +1,346 @@
...
+| used                  | Storage usages. Unit : B |
+
+
+<!--Traffic-->
+
+| Metric name                         | Description                                                      |
+| :---------------------------------- | :--------------------------------------------------------------- |
+| status                              | Status of the interface                                          |
+| interface.traffic.in.bitspersecond  | Incoming traffic going through the interface. Units: B/s & %     |
+| interface.traffic.out.bitspersecond | Outgoing traffic going through the interface. Units: B/s & %     |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:181]:
```suggestion
| interface.traffic.in.bitspersecond  | Incoming traffic going through the interface. Units: B/s     |
| interface.traffic.out.bitspersecond | Outgoing traffic going through the interface. Units: B/s    |
```

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 9]
```diff
@@ -0,0 +1,346 @@
...
+	--verbose \
+	--use-new-perfdata
+	
+	
+OK: CPU(s) average usage is 11.91 % - CPU '0' usage : 11.91 % |
+'cpu.utilization.percentage'=11.91%;;;0;100 '0#core.cpu.utilization.percentage'=11.91%;;;0;100 
+CPU '0' usage : 11.91 %
+```
+
+The command above gets the average of a Linux SSH CPU (```--mode=CPU```).
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:331]: Maybe a better explanation of the SSH options.

**Review by pkriko — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md (hunk 9 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:331]: Adding explanation in below lines

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md (hunk 5 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md]: Adding all perfdatas for cpu-detailed

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md (hunk 7 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md]: Done ^^

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md (hunk 8 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md]: done

File: en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+    --use-new-perfdata
+		
+OK: CPU(s) average usage is 11.91 % - CPU '0' usage : 11.91 % |
+'cpu.utilization.percentage'=11.91%;;;0;100 '0#core.cpu.utilization.percentage'=11.91%;;;0;100 
+CPU '0' usage : 11.91 %
+```
+
+The check command gives the average CPU of a device using SSH (```--mode=CPU```) 
+It has IP address 10.30.2.114 (```--hostname=10.30.2.114```) as backend (```--ssh-backend='libssh'```) with the centeron as username _centreon_ (```--ssh-username=centreon```) 
+and as password _centreon-password_ (```--ssh-password='centreon-password'```).
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md:338]: Could you reformulate ?

**Review by pkriko — APPROVED** [review]

**Pull request merged**